src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -6,16 +6,6 @@
 
 var MobileCarousel = require( '../../../modules/classes/MobileCarousel' );
 
-function init() {
-
-  // TODO: Remove this when per-page JS is introduced.
-  if ( document.querySelectorAll( '.careers-working-at-cfpb' ).length === 0 ) {
-    return;
-  }
-
-  // MobileCarousel with 599px breakpoint.
-  var mobileCarousel = new MobileCarousel( 599 );
-  mobileCarousel.enableOn( '.js-mobile-carousel' );
-}
-
-module.exports = { init: init };
+// MobileCarousel with 599px breakpoint.
+var mobileCarousel = new MobileCarousel( 599 );
```

Comment:
You shouldn't have to pass any value. It should default to [599](https://github.com/cfpb/cfgov-refresh/blob/flapjack/src/static/js/modules/classes/MobileCarousel.js)?